packages/web/src/components/user-badges/FollowsYouBadge.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import styles from './FollowsYouBadge.module.css'
+
+export default function FollowsYouBadge() {
+  return <div className={styles.badge}>FOLLOWS YOU</div>
```

Comment:
we are following a convention to put all text in a `messages` object at the top of each file so it'll be easier for i18n at some point. so something like `const messages = {followsYouText: 'follows you'}`

Also we are trying to avoid UPPERCASE text since some screenreaders read out each letter :D. The way we avoid this is adding `text-transform: uppercase;` to its styles.

And one last minor thing, does it make sense to have this as a span so that it can stay inline with the h2 text?